### PR TITLE
[reinvent] optimized extension

### PR DESCRIPTION
**Pull request by cswatt**
### What does this PR do? What is the motivation?
Docs changes for optimized extension GA
ready to merge whenever

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/serverless/aws_lambda/installation/go.md [hunk 1]
```diff
@@ -157,6 +157,11 @@ func myHandler(ctx context.Context, _ events.APIGatewayProxyRequest) (string, er
 }
 ```
 
+## Minimize cold start duration
+Version 67+ of [the Datadog Extension][5] is optimized to reduce instrumentation overhead by up to 70%. To opt out of the optimized extension...
```
Comment (sumedham) [inline content/en/serverless/aws_lambda/installation/go.md:161]:
```suggestion
Version 67+ of [the Datadog Extension][5] is optimized to reduce instrumentation overhead by up to 70%. 

If you are noticing any issues, you can force your extension to use the fully compatible, older version of the extension by setting `DD_EXTENSION_VERSION` to `compatibility`. We also encourage reporting any feedback or bugs by adding an [issue on GitHub][6] and tag your issue with `version/next`.
```

File: content/en/serverless/aws_lambda/installation/go.md [hunk 2]
```diff
@@ -157,6 +157,11 @@ func myHandler(ctx context.Context, _ events.APIGatewayProxyRequest) (string, er
...
+
+To leave feedback or report a bug, add an [issue on GitHub][6] and tag your issue with `version/next`.
```
Comment (sumedham) [inline content/en/serverless/aws_lambda/installation/go.md:163]:
```suggestion

```

File: content/en/serverless/aws_lambda/installation/go.md (hunk 1 above)
Comment (sumedham) [reply content/en/serverless/aws_lambda/installation/go.md:161]: can we add this for all files

**Comment by sumedham** [comment]
I know we have a few different callouts at the very top. but can we add a new one with the following

`Version 67+ of the Datadog Lambda Extension uses an optimized version of the extension. Read more.`. 

if we have too many, I'm ok with moving the forwarder migration and the privatelink ones into minor sections on the same page.

**Review by sumedham — CHANGES_REQUESTED** [review]

File: content/en/serverless/aws_lambda/installation/go.md [hunk 3]
```diff
@@ -157,6 +159,11 @@ func myHandler(ctx context.Context, _ events.APIGatewayProxyRequest) (string, er
 }
 ```
 
+## Minimize cold start duration
+Version 67+ of [the Datadog Extension][5] is optimized to reduce instrumentation overhead by up to 70%.
```
Comment (sumedham) [inline content/en/serverless/aws_lambda/installation/go.md:163]:
```suggestion
Version 67+ of [the Datadog Extension][5] is optimized to significantly reduce cold start duration.
```

File: content/en/serverless/aws_lambda/installation/java.md [hunk 4]
```diff
@@ -365,6 +367,11 @@ module "lambda-datadog" {
 {{% /tab %}}
 {{< /tabs >}}
 
+## Minimize cold start duration
+Version 67+ of [the Datadog Extension][12] is optimized to reduce instrumentation overhead by up to 70%.
```
Comment (sumedham) [inline content/en/serverless/aws_lambda/installation/java.md:371]:
```suggestion
Version 67+ of [the Datadog Extension][12] is optimized to significantly reduce cold start duration.
```

File: content/en/serverless/aws_lambda/installation/python.md [hunk 5]
```diff
@@ -402,8 +404,10 @@ module "lambda-datadog" {
 {{% /tab %}}
 {{< /tabs >}}
 
-## Minimize Cold Start Duration (Beta)
-Starting with version 63 of [the Datadog Extension][7], you can set the environment variable `DD_EXTENSION_VERSION` to `next` to use an optimized version of the Datadog Extension that reduces instrumentation overhead by up to 70%. To leave feedback or report a bug, please add an [issue on Github][8] and tag your issue with `version/next`.
+## Minimize cold start duration
+Version 67+ of [the Datadog Extension][7] is optimized to reduce instrumentation overhead by up to 70%.
```
Comment (sumedham) [inline content/en/serverless/aws_lambda/installation/python.md:408]:
```suggestion
Version 67+ of [the Datadog Extension][7] is optimized to significantly reduce cold start duration.
```

File: content/en/serverless/aws_lambda/installation/ruby.md [hunk 6]
```diff
@@ -328,6 +330,11 @@ To install and configure the Datadog Serverless Plugin, follow these steps:
 {{% /tab %}}
 {{< /tabs >}}
 
+## Minimize cold start duration
+Version 67+ of [the Datadog Extension][10] is optimized to reduce instrumentation overhead by up to 70%.
```
Comment (sumedham) [inline content/en/serverless/aws_lambda/installation/ruby.md:334]:
```suggestion
Version 67+ of [the Datadog Extension][10] is optimized to significantly reduce cold start duration.
```

File: content/en/serverless/aws_lambda/installation/nodejs.md [hunk 7]
```diff
@@ -379,8 +381,10 @@ module "lambda-datadog" {
 
 <div class="alert alert-warning">Do not install the Datadog Lambda Library as a layer <i>and</i> as a JavaScript package. If using the Datadog Lambda Layer (recommended), do not include <code>datadog-lambda-js</code> in your <code>package.json</code>, or install it as a dev dependency and run <code>npm install --production</code> before deploying.</div>
 
-## Minimize Cold Start Duration (Beta)
-Starting with version 63 of [the Datadog Extension][7], you can set the environment variable `DD_EXTENSION_VERSION` to `next` to use an optimized version of the Datadog Extension that reduces instrumentation overhead by up to 70%. To leave feedback or report a bug, add an [issue on GitHub][8] and tag your issue with `version/next`.
+## Minimize cold start duration
+Version 67+ of [the Datadog Extension][7] is optimized to reduce instrumentation overhead by up to 70%.
```
Comment (sumedham) [inline content/en/serverless/aws_lambda/installation/nodejs.md:385]:
```suggestion
Version 67+ of [the Datadog Extension][7] is optimized to significantly reduce cold start duration.
```

File: content/en/serverless/aws_lambda/installation/dotnet.md [hunk 8]
```diff
@@ -298,6 +300,11 @@ module "lambda-datadog" {
 {{% /tab %}}
 {{< /tabs >}}
 
+## Minimize cold start duration
+Version 67+ of [the Datadog Extension][7] is optimized to significantly reduce cold start duration.
+
+If you are noticing any issues, you can force your extension to use the fully compatible, older version of the extension by setting `DD_EXTENSION_VERSION` to `compatibility`. Datadog also encourages you to report any feedback or bugs by adding an [issue on GitHub][8] and tag your issue with `version/next`.
```
Comment (jhgilbert) [inline content/en/serverless/aws_lambda/installation/dotnet.md:306]:
```suggestion
If you notice any issues, you can force your extension to use the fully compatible, older version of the extension by setting `DD_EXTENSION_VERSION` to `compatibility`. Datadog also encourages you to report any feedback or bugs by adding an [issue on GitHub][8] and tagging your issue with `version/next`.
```

File: content/en/serverless/aws_lambda/installation/go.md [hunk 9]
```diff
@@ -157,6 +159,11 @@ func myHandler(ctx context.Context, _ events.APIGatewayProxyRequest) (string, er
 }
 ```
 
+## Minimize cold start duration
+Version 67+ of [the Datadog Extension][5] is optimized to significantly reduce cold start duration.
+
+If you are noticing any issues, you can force your extension to use the fully compatible, older version of the extension by setting `DD_EXTENSION_VERSION` to `compatibility`. Datadog also encourages you to report any feedback or bugs by adding an [issue on GitHub][6] and tag your issue with `version/next`.
```
Comment (jhgilbert) [inline content/en/serverless/aws_lambda/installation/go.md:165]:
```suggestion
If you notice any issues, you can force your extension to use the fully compatible, older version of the extension by setting `DD_EXTENSION_VERSION` to `compatibility`. Datadog also encourages you to report any feedback or bugs by adding an [issue on GitHub][6] and tagging your issue with `version/next`.
```

File: content/en/serverless/aws_lambda/installation/java.md [hunk 10]
```diff
@@ -365,6 +367,11 @@ module "lambda-datadog" {
 {{% /tab %}}
 {{< /tabs >}}
 
+## Minimize cold start duration
+Version 67+ of [the Datadog Extension][12] is optimized to significantly reduce cold start duration.
+
+If you are noticing any issues, you can force your extension to use the fully compatible, older version of the extension by setting `DD_EXTENSION_VERSION` to `compatibility`. Datadog also encourages you to report any feedback or bugs by adding an [issue on GitHub][13] and tag your issue with `version/next`.
```
Comment (jhgilbert) [inline content/en/serverless/aws_lambda/installation/java.md:373]:
```suggestion
If you notice any issues, you can force your extension to use the fully compatible, older version of the extension by setting `DD_EXTENSION_VERSION` to `compatibility`. Datadog also encourages you to report any feedback or bugs by adding an [issue on GitHub][13] and tagging your issue with `version/next`.
```

File: content/en/serverless/aws_lambda/installation/nodejs.md [hunk 11]
```diff
@@ -379,8 +381,10 @@ module "lambda-datadog" {
 
 <div class="alert alert-warning">Do not install the Datadog Lambda Library as a layer <i>and</i> as a JavaScript package. If using the Datadog Lambda Layer (recommended), do not include <code>datadog-lambda-js</code> in your <code>package.json</code>, or install it as a dev dependency and run <code>npm install --production</code> before deploying.</div>
 
-## Minimize Cold Start Duration (Beta)
-Starting with version 63 of [the Datadog Extension][7], you can set the environment variable `DD_EXTENSION_VERSION` to `next` to use an optimized version of the Datadog Extension that reduces instrumentation overhead by up to 70%. To leave feedback or report a bug, add an [issue on GitHub][8] and tag your issue with `version/next`.
+## Minimize cold start duration
+Version 67+ of [the Datadog Extension][7] is optimized to significantly reduce cold start duration.
+
+If you are noticing any issues, you can force your extension to use the fully compatible, older version of the extension by setting `DD_EXTENSION_VERSION` to `compatibility`. Datadog also encourages you to report any feedback or bugs by adding an [issue on GitHub][8] and tag your issue with `version/next`.
```
Comment (jhgilbert) [inline content/en/serverless/aws_lambda/installation/nodejs.md:387]:
```suggestion
If you notice any issues, you can force your extension to use the fully compatible, older version of the extension by setting `DD_EXTENSION_VERSION` to `compatibility`. Datadog also encourages you to report any feedback or bugs by adding an [issue on GitHub][8] and tagging your issue with `version/next`.
```

File: content/en/serverless/aws_lambda/installation/python.md [hunk 12]
```diff
@@ -402,8 +404,10 @@ module "lambda-datadog" {
 {{% /tab %}}
 {{< /tabs >}}
 
-## Minimize Cold Start Duration (Beta)
-Starting with version 63 of [the Datadog Extension][7], you can set the environment variable `DD_EXTENSION_VERSION` to `next` to use an optimized version of the Datadog Extension that reduces instrumentation overhead by up to 70%. To leave feedback or report a bug, please add an [issue on Github][8] and tag your issue with `version/next`.
+## Minimize cold start duration
+Version 67+ of [the Datadog Extension][7] is optimized to significantly reduce cold start duration.
+
+If you are noticing any issues, you can force your extension to use the fully compatible, older version of the extension by setting `DD_EXTENSION_VERSION` to `compatibility`. Datadog also encourages you to report any feedback or bugs by adding an [issue on GitHub][8] and tag your issue with `version/next`.
```
Comment (jhgilbert) [inline content/en/serverless/aws_lambda/installation/python.md:410]:
```suggestion
If you notice any issues, you can force your extension to use the fully compatible, older version of the extension by setting `DD_EXTENSION_VERSION` to `compatibility`. Datadog also encourages you to report any feedback or bugs by adding an [issue on GitHub][8] and tagging your issue with `version/next`.
```

File: content/en/serverless/aws_lambda/installation/ruby.md [hunk 13]
```diff
@@ -328,6 +330,11 @@ To install and configure the Datadog Serverless Plugin, follow these steps:
 {{% /tab %}}
 {{< /tabs >}}
 
+## Minimize cold start duration
+Version 67+ of [the Datadog Extension][10] is optimized to significantly reduce cold start duration.
+
+If you are noticing any issues, you can force your extension to use the fully compatible, older version of the extension by setting `DD_EXTENSION_VERSION` to `compatibility`. Datadog also encourages you to report any feedback or bugs by adding an [issue on GitHub][11] and tag your issue with `version/next`.
```
Comment (jhgilbert) [inline content/en/serverless/aws_lambda/installation/ruby.md:336]:
```suggestion
If you notice any issues, you can force your extension to use the fully compatible, older version of the extension by setting `DD_EXTENSION_VERSION` to `compatibility`. Datadog also encourages you to report any feedback or bugs by adding an [issue on GitHub][11] and tagging your issue with `version/next`.
```

**Review by jhgilbert — APPROVED** [review]
Approved with very minor suggestions, thanks!

**Pull request merged**